From de5c8902a166f08103ea73a2d4becef2c6f479ac Mon Sep 17 00:00:00 2001
From: Andra-Teodora Ilie <andra.ilie@nxp.com>
Date: Tue, 5 Apr 2022 12:37:17 +0300
Subject: [PATCH 081/103] Revert "dspi: Add support for 8 and 16 bit frame
 size"

This reverts commit fb64137820519b311517b18948f00ff9ed955def.

Issue: ALB-8598
Upstream-Status: Pending 

Signed-off-by: Andra-Teodora Ilie <andra.ilie@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/spi/fsl_dspi.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/spi/fsl_dspi.c b/drivers/spi/fsl_dspi.c
index 071a6f8502..d368848c5b 100644
--- a/drivers/spi/fsl_dspi.c
+++ b/drivers/spi/fsl_dspi.c
@@ -263,10 +263,6 @@ static int fsl_dspi_cfg_ctar_mode(struct fsl_dspi_priv *priv,
 		bus_setup |= DSPI_CTAR_CPHA;
 	if (mode & SPI_LSB_FIRST)
 		bus_setup |= DSPI_CTAR_LSBFE;
-	if (mode & SPI_FMSZ_8)
-		bus_setup |= DSPI_CTAR_TRSZ(7);
-	if (mode & SPI_FMSZ_16)
-		bus_setup |= DSPI_CTAR_TRSZ(15);
 
 	ns_delay_scale(&pcssck, &cssck, CONFIG_DSPI_CS_SCK_DELAY,
 		       priv->bus_clk);
-- 
2.25.1

